lib/matplotlib/path.py

```diff
@@ -517,6 +521,17 @@ def contains_point(self, point, transform=None, radius=0.0):
         Returns
         -------
         bool
+
+        Notes
+        -----
+        The current algorithm has some limitations:
+
+        - The result is undefined for points exactly at the boundary
+          (including *radius*).
```

Comment:
This makes it sound like if you make radius larger, then the area which is not defined increases, which is not true. It's just wherever the boundary ends up being the result is undefined there, unless I'm mistaken?

Reply:
I'd say the new version is clear now.